frontend/src/components/layout/navigation/MobileNavigation.module.scss

```diff
@@ -3,7 +3,9 @@
 
 .mobile-menu {
   width: 100%;
-  position: relative;
+  position: sticky;
+  top: 76px; // threshold for mobile menu - height of header
+  height: 0; // prevents element from pushing main elements down
```

Comment:
This is a good comment; it answers something I'd definitely wonder about if it wasn't there :)